Technical-Steering-Committee/proposal.md

```diff
@@ -0,0 +1,89 @@
+
+This proposal is to form a Zowe Technical Steering Committee (TSC) to oversee, govern, and improve the infrastructure, execution, and customer experience related to development, testing, packaging, delivery, and cross-squad technology overlap areas.  
+
+## Background
+
+When Zowe was first created different technology from different codebase combined and was stewarded by individual squads who oversaw each component.  Each squad was responsible for tracking and planning the backlog of items for their component.
+
+As Zowe has grown, coordination between squads has grown, resulting in a common installer for z/OS components, and more recently features such as Sign Sign On (SSO), MultiFactor Authentication (MFA) and client-side certificates have required technical and delivery coordination between the different components. These capabilities have enhanced Zowe’s value to the community.  It is anticipated this vector of integration will continue in the future with high availability (HA), containerization, improved security and other cross component features.  
+
+The coordination and synchronization of the moving parts of Zowe is becoming more central to successful delivery of features and for this reason the Zowe Leadership Committee (ZLC) taking leadership guidance from other successful Linux Foundation projects is proposing the creation a Technical Steering Committee (TSC).
+
+## Role
+
+The TSC will oversee the technical direction of the Zowe project. Responsibilities include, but are not limited to
+
+- Setting the release schedule, release quality standards, and approving releases.
+- Maintaining the overall project governance and processes.
+- Coordinating the use of project infrastructure, include code repositories, issue tracking, CI/CD environments, and communication tools.
+- Establish a Code of Conduct inline with the Open Mainframe Project and Linux Foundation standards.
+- Maintaining the list of committers.
+- Establish and maintain the overall development processes, including any coding, testing, security, and best practice standards for squads to operate within.
+- Faciliate cross-squad interactions, including cross-squad efficiencies, resource management, and balancing the development focuses between technical debt versus new content delivery.
+- Establish and maintain a lifecycle for Zowe squads, serving as a mentor to newly established squads.
+- Encourage participation and mentorship in wider community events such as hackathons or mentorships.
+- Appointing representatives to work with other open source or open standards communities as needed.
+
+## Composition
+
+The TSC is intended to be a representation of Zowe Committers, as well as other architects, subject matter experts, and members of the extended Zowe community. While any of these individuals may participate in the TSC, there will be a defined group of TSC voting members that will vote on actions of the TSC within the roles of the TSC outlined above.
```

Comment:
I re-worded this a bit to make it clear who the TSC members are and that it is a body of the community. This also aligns with how other LF and OMP projects look at this.